torch/_subclasses/fake_tensor.py

```diff
@@ -860,6 +867,15 @@ def merge_devices(t: object) -> None:
 
         return common_device, has_scalar_only_inputs
 
+    def get_nested_int(
+        self,
+        *,
+        coeff: Union[int, torch.SymInt] = 1,
+    ) -> torch.SymInt:
+        if self.nested_int_memo is None:
```

Comment:
do we ever want this to be None?

Reply:
We expect it to be None when an offsets is a graph intermediate, or an input that has never been associated with a nested int, and then that offsets is later used during NT construction.

Reply:
do you mind adding that explanation in a comment near where the memo is defined?